packages/core-components/src/layout/HeaderTabs/HeaderTabs.tsx

```diff
@@ -108,14 +133,16 @@ export function HeaderTabs(props: HeaderTabsProps) {
         value={selectedTab}
       >
         {tabs.map((tab, index) => (
-          <TabUI
+          <LinkTab
             {...tab.tabProps}
+            href={window.location.href}
```

Comment:
I don't think `window.location.href` is what you are looking for here.

Reply:
how to pass the path then ?
```
import Link from 'backstage/component'
<tabui 
component={Link}
/>
```
if i pass to = {tab.id} it doesn't append path after `catalog/tab.id` path it directs to `localhost/tab.id` 
I have some ideas but any suggestions?